### DISCUSSION TITLE
Text input and output transformations

### DISCUSSION THREAD

**Discussion by briansunter** [briansunter/logseq-plugin-gpt3-openai#184] [Ideas]
Allow users to process text input before sending it to gpt3 and process the output, using regex and javascript

For example, extracting all highlights on the page or blocks with a certain tag.

Or processing the output, such as splitting up a list into multiple blocks.

**Comment by AvijeetPrasad** [comment]
I would be nice for us to be able to edit the text returned by OpenAI in the pop-up window before we press insert.
